src/scripts/main.js

```diff
@@ -1,3 +1,92 @@
 'use strict';
 
-// write code here
+const buttonContainer = document.querySelector('.container');
+const maxRow = 10;
+const minRow = 2;
+const maxColumn = 10;
+const minColumn = 2;
+
+buttonContainer.addEventListener('click', (e) => {
+  const target = e.target;
+  const tbody = document.querySelector('.field tbody');
+  const className = target.className.slice(0, -7);
```

Comment:
if interesting, you can read this article and add some improvements to controls

https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes

it will add a better code understanding and prevent bugs if someone change className on element

Reply:
yes, I know about data attributes and use them in my projects

Reply:
При выполнении скрипта мы попадем лишь в один какой-нибудь кейс из четырех и, как следствие, будем искать в дом-дереве только один элемент. В варианте, который предлагаешь ты, будут искаться сразу все четыре элемента. Зачем это делать? В чем логика?